less/index.less

```diff
@@ -52,8 +52,126 @@ table {
 * {
     box-sizing: border-box;
 }
+// Variables
+@background-color: @Gallery;
+@Gallery: #F0F0F0;
+
+@h2-font-size: 2.5rem;
+@p-font-size: 1rem;
 
 html, body {
+	background: @Gallery;
     height: 100%;
 }
+.container-home {
+	max-width: 1200px;
+	width: 100%;
+	margin: 0 auto;
+
+	.header {
+		margin: 1% 0 2% 0;
+
+		h1 {
+		display: flex;
+		justify-content: center;
+		font-size: 3rem;
+	
+		}
+		.nav {
+		display: flex;
+		flex-direction: row;
+		width: 100%;
+		justify-content: space-between;
+		font-size: 1.2rem;
+		
+		a {
+			color: black;
+			text-decoration: none;
+			
+		}	
+			a:hover {
+			font-size: 1.7rem;
+			color:black;
+			}
+		}
+}
+
+.dash {
+	margin: 1% 0 1% 0;
+	border: 1px solid black;
+}
+.image {
+	// border: 3px solid black;
+	display: flex;
+	justify-content: center;
+	width: 100%;
+	height: 200px;
+	
+	img {
+		width: 50%;
+	}
+}
+}//container-home
```

Comment:
Great convention to comment the end of a wrapping container div. It makes your code much more readable than without it. Look into a VS Code extension that will tidy up your formatting on save, this way, the spacing and indentation can automatically be formatted, making your code even _more_ readable.